config/default.yml

```diff
@@ -2234,6 +2234,13 @@ Rails/ActiveRecordAliases:
   Enabled: true
   VersionAdded: '0.53'
 
+Rails/ActiveRecordOverride:
+  Description: >-
+                  Check for overriding Active Record methods instead of using
+                  callbacks.
+  Enabled: true
+  VersionAdded: '0.67'
```

Comment:
Don't forget to limit the cop to `app/controllers/**/*.rb`.

Reply:
It should actually be `app/models/**/*.rb`.